.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
   "parser": "babel-eslint",
   'env': {
-    'es6': true
+    'es6': true,
+    'node': true
```

Comment:
It's not all node code (and you specify this in this file and in the client one)

Can you specify the client as browser and commonjs, and the api as node?